src/jti_background.js

```diff
@@ -213,6 +254,80 @@ function addTemplate (templateName, issueTypeField, projectsField, text, callbac
     });
 }
 
+function addDomain (domainName, callback) {
+    chrome.storage.sync.get(StorageID, function (data) {
+        var domainJSON = {};
+
+        if (data[StorageID]) {
+            domainJSON = data[StorageID];
+        }
+
+        var domainID = getNextID(domainJSON.options.domains);
+        var newDomain = {
+            'id': domainID,
+            'name': domainName
+        };
+
+        validateDomain(domainName, function (message) {
+            if (message) {
+                callback(false, message);
+            } else {
+                domainJSON.options.domains[domainID] = newDomain;
+                // Refresh existing pages with this URL.
+                matchRegexToJsRegex(domainName);
+                chrome.tabs.query({windowId: chrome.windows.WINDOW_ID_CURRENT}, function (tabs) {
+                    $.each(tabs, function (tabIndex, tab) {
+                        var chromeRegex = new RegExp('chrome://extensions');
```

Comment:
When you add a domain, we refresh the page. If you refresh the chrome://extensions page it refreshes the JTI. On refresh, the JTI reloads all the pages that match its domains, including the chrome://extensions page, hence an infinite reload loop.

I feel that this will never be an issue, I don't think anyone will need or want to look for Jira fields in chrome://extensions.